### PR TITLE
Daily PR April 19

### PR DESCRIPTION
**Pull request by pe7r**
Work with PCard styles for tablet and mobile.

### CODE REVIEW

File: src/Components/Products/Products.css [hunk 1]
```diff
@@ -35,24 +35,69 @@
...
 }
 
-@media screen and (max-width: 767px) {
+@media (max-width: 1024px) {
     .products__col {
         display: flex;
         flex-direction: column;
         min-height: 230px;
-        width: 48%;
+        width: 30.8%;
```
Comment (ruslanpalagin) [inline src/Components/Products/Products.css]: Replace with **width: 33.333%**
adjust css rules logic to fit width

File: src/Components/Products/Products.css [hunk 2]
```diff
@@ -35,24 +35,69 @@
...
+        padding: 0px 10px;
+    }
+}
+
+@media screen and (max-width: 767px) {
+    .products__col {
+        display: flex;
+        flex-direction: column;
+        min-height: 230px;
+        width: 46.71%;
```
Comment (ruslanpalagin) [inline src/Components/Products/Products.css]: width must be 50%

File: src/Components/Products/Products.css [hunk 3]
```diff
@@ -35,24 +35,69 @@
...
+        justify-content: space-around;
+    }
+}
+
+@media (max-width: 374px) {
+    .products__col {
+        display: flex;
+        flex-direction: column;
+        min-height: 230px;
+        width: 65%;
```
Comment (ruslanpalagin) [inline src/Components/Products/Products.css]: width must be 100%;
you should limit width with container

File: src/Components/Products/Products.css (hunk 1 above)
Comment (pe7r) [reply src/Components/Products/Products.css]: Fixed.

File: src/Components/Products/Products.css (hunk 3 above)
Comment (pe7r) [reply src/Components/Products/Products.css]: Fixed.

File: src/Components/Products/Products.css (hunk 2 above)
Comment (pe7r) [reply src/Components/Products/Products.css]: Fixed.

**Pull request merged**